packages/models/src/bitcoin-unit.model.ts

```diff
@@ -0,0 +1,46 @@
+import BigNumber from 'bignumber.js';
+
+export enum WalletBitcoinUnit {
+  bitcoin = 'bitcoin',
+  millibit = 'millibit',
+  bit = 'bit',
+  satoshi = 'satoshi',
+  millisatoshi = 'millisatoshi',
+}
+export type BitcoinUnit = keyof typeof WalletBitcoinUnit;
+
+export type BitcoinUnitSymbol = 'BTC' | 'mBTC' | 'μBTC' | 'sat' | 'msat';
+
+export interface BitcoinUnitInfo {
+  name: WalletBitcoinUnit;
+  symbol: BitcoinUnitSymbol;
+  decimal: BigNumber;
+}
+
+export const bitcoinUnitsKeyedByName: Record<WalletBitcoinUnit, BitcoinUnitInfo> = {
+  [WalletBitcoinUnit.bitcoin]: {
+    name: WalletBitcoinUnit.bitcoin,
+    symbol: 'BTC',
+    decimal: BigNumber(1),
+  },
+  [WalletBitcoinUnit.millibit]: {
+    name: WalletBitcoinUnit.millibit,
+    symbol: 'mBTC',
+    decimal: BigNumber(0.001),
```

Comment:
@kyranjamie using here, this seem ok?

Reply:
wouldn't this be something for constant pkg?

Reply:
I can move there ...I forgot the `new` syntax btw, haha.